buildenv

```diff
@@ -40,5 +43,5 @@ zopen_post_install()
 
 zopen_get_version()
 {
-  echo "9.3"
+  echo "$OPENSSH_VERSION"
```

Comment:
I see the build failed due to an error. I think we need to strip the p1 for now since zopen_get_version is expecting an number for the minor release.

Reply:
Done